Gemfile

```diff
@@ -11,7 +11,7 @@ gem 'charlock_holmes', '~> 0.6.9.4'
 gem 'dynamic_form', '~> 1.1.4'
 gem 'exception_notification', '~> 3.0.1'
 gem 'fancybox-rails', '~> 0.2.1'
-gem 'foundation-rails', '~> 5.5.3.2'
+gem 'foundation-rails', '~> 5.5.3.2' # 6.x.x.x versions cause problems (see blame)
```

Comment:
Don't really need the comment due to the nice commit message. I know I added one in the past, but I think now we've done a few of these its already convention to check the blame for why a gem might be pinned at a particular version.